src/cmd_line/subparser.ts

```diff
@@ -225,6 +226,11 @@ export const commandParsers = {
     abbrev: 'xa',
     parser: parseWriteQuitAllCommandArgs,
   },
+
+  put: {
```

Comment:
This was recently changed to be sorted - could you put this command in the right place please?

Reply:
Fixed!